frontend/src/Hooks/useGameState.js

```diff
@@ -315,7 +312,7 @@ export default function useGameState() {
             dispatch({
                 type: 'trial-end',
                 status:
-                    playerKilled === 'abstainVote' || !playerKilled
+                    playerKilled === 'abstain Vote' || !playerKilled
```

Comment:
Made an issue to make this an enum: https://github.com/Mafia-Hands/Mafia/issues/230